pkg/amphoracontrollers/daemonset.go

```diff
@@ -90,6 +90,18 @@ func DaemonSet(
 		volumeMounts = append(volumeMounts, instance.Spec.TLS.CreateVolumeMounts(nil)...)
 	}
 
+	// Because we don't use jobboard, we need to ensure that the octavia
+	// controllers are gracefully shutdown, so after they receive the signal,
+	// they need to complete the jobs that are being executed (creating a LB,
+	// updating a listener)
+	// 630 sec is close to the value that was used in tripleo, it's based on the
```

Comment:
This says `630` but then `600` is used below.  Is this a typo?

Reply:
oops sorry, yeah it should be 600 sec. I hesitated for a long time :D

Reply:
updated with 600